docs/specs/output.yml

```diff
@@ -672,6 +672,13 @@ outputs:
     {"message_severity":"error","code":"schema-not-found","message":"Unknown schema ''","file":"a/original.xml"}
   .dependencymap.json: |
     {"dependencies":{"a/b.original.xml":[{"source":"a/original.xml","type":"file"}]}}
+  .publish.json: |
+    {
+      "files": [
+        {"url": "/b", "source_path": "a/b.original.xml"},
```

Comment:
Would you add a comment to `PublishItem.SourcePath` to say who is using it? I couldn't remember it immediately and digged into code, it seems to be used by publish output for PR comment?

Reply:
yes